Casks/s/swiftformat-for-xcode.rb

```diff
@@ -1,12 +1,31 @@
 cask "swiftformat-for-xcode" do
-  version "0.57.0"
-  sha256 "cb677b9cdd835ef1484129866adfdfd0e06118883d091407f64d79e7b443bd61"
+  version "0.57.1"
+  sha256 "9201310287a0818f14f5fd96003a575132a98a53f965add2976698abf47d6ed7"
 
-  url "https://github.com/nicklockwood/SwiftFormat/releases/download/#{version}/SwiftFormat.for.Xcode.app.zip"
+  url "https://github.com/nicklockwood/SwiftFormat/releases/download/#{version}/SwiftFormat.for.Xcode.zip"
   name "SwiftFormat for Xcode"
   desc "Xcode Extension for reformatting Swift code"
   homepage "https://github.com/nicklockwood/SwiftFormat"
 
+  # Not every GitHub release provides a file for macOS or each arch
+  # so we check multiple recent releases instead of only the "latest" release.
+  livecheck do
+    url :url
+    regex(/SwiftFormat[._-]for[._-]Xcode(?:\.app)?\.zip/i)
+    strategy :github_releases do |json, regex|
+      json.map do |release|
+        next if release["draft"] || release["prerelease"]
+
+        release["assets"]&.map do |asset|
+          match = asset["name"]&.match(regex)
+          next if match.blank?
+
+          release["tag_name"]
```

Comment:
```suggestion
          release["tag_name"]&.[](/^v?(\d+(?:\.\d+)+)$/i, 1)
```

It might make sense to match versions using a regex to stay in line with the default behaviour of the `GithubReleases` strategy. This suggestion uses the default regex.